core/utils/device/adb.py

```diff
@@ -124,23 +142,51 @@ def get_page_source(device_id):
             # In such cases return empty string.
             return ''
 
-    # noinspection PyPep8Naming
     @staticmethod
     def is_text_visible(device_id, text, case_sensitive=False):
+        element = Adb.get_element_by_text(device_id, text, case_sensitive)
+        return element is not None
+
+    @staticmethod
+    def convert_element_bounds_to_x_y_coordinates(element):
```

Comment:
What about naming it `get_coordinates(element)` or something else which is shorter.

Reply:
done